packages/polaris-viz/src/components/SwallowErrors/SwallowErrors.tsx

```diff
@@ -0,0 +1,31 @@
+import {Component} from 'react';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+interface State {
+  hasError: boolean;
+}
+
+export class SwallowErrors extends Component<Props, State> {
```

Comment:
Maybe I'm not quite sure how these work. It seems to be that this isn't used anywhere?